app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -30,9 +30,62 @@ interface FormattedLocation {
   weekdayHours: string;
 }
 
+interface PostOptions {
+  /**
+   * Whether to send the request as a JSON request. Defaults to inclusion.
+   */
+  json: boolean;
+
+  /**
+   * Whether to include CSRF token in the request. Defaults to inclusion.
+   */
+  csrf: boolean;
+
+  /**
+   * Optional. HTTP verb used. If excluded, will fetch as a GET.
+   */
+  method: string;
+}
+
+const getCsrfToken = () =>
```

Comment:
Super nit: Personally I prefer each letter of an acronym to be capitalized for consistency with other code and with common web API naming conventions (except [XMLHttpRequest](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest), which I prefer to pretend doesn't exist).

I could see a counter-point being that this would be most consistent with our Ruby code, so I'm not going to hold too strong to this opinion.

```suggestion
const getCSRFToken = () =>
```